fbw-a32nx/src/systems/instruments/src/SD/Pages/Press/Press.tsx

```diff
@@ -54,6 +54,7 @@ export const PressPage: FC = () => {
   }, [cabinAlt]);
 
   const deltaPress = splitDecimals(MathUtils.clamp(deltaPsi, -9.9, 9.9));
+  const deltaPressNotAvail = arincDeltaPsi.isNoComputedData();
```

Comment:
Normally the DMC should use the cabin pressure subtracted by the corrected static pressure as received directly from the ADR, as a backup if the delta pressure field is not available. I'm not entirely sure which failure case this is supposed to cover, as (outside of an isolated ARINC transceiver failure on the CPCs or something) the delta pressure field should be available when both the CPC and the ADR is functioning normally. Maybe something to add as a comment, but not critical

Reply:
Yeah I see what you mean. So the "XX" would only appear when the data from the SDAC is not valid (which, in this case, would be when the CPC is not transmitting delta pressure AND the ADR is not transmitting static ambient pressure).

Since the SDAC is not modelled I think this logic is a good placeholder for now as the behaviour is correct, but I've added a comment as suggested. Thanks!